cmd/_integration-tests/transport/http_test.go

```diff
@@ -158,6 +158,28 @@ func TestCtxToCtxViaHTTPHeaderClient(t *testing.T) {
 	}
 }
 
+// Test different kinds of message field names from protobuf definition.
+// e.g. "CamelCase", "snake_case", "__why_so_many_underscores"
+func TestEchoOddNamesClient(t *testing.T) {
+	var req pb.OddFieldNames
```

Comment:
This is just a stylistic thing, and not something I care if you change, but more typically, I think, are single-statement initializations, versus multiple-statement initializations like so:
```go
req := pb.OddFieldNames{
    CamelCase: 12,
    SnakeCase: 24,
    XWhy_So_Many_Underscores: 36,
}
```